server/src/test/java/org/elasticsearch/index/query/GeoDistanceQueryBuilderTests.java

```diff
@@ -354,4 +369,21 @@ public void testParseFailsWithMultipleFields() throws IOException {
         ParsingException e = expectThrows(ParsingException.class, () -> parseQuery(json));
         assertEquals("[geo_distance] query doesn't support multiple fields, found [point1] and [point2]", e.getMessage());
     }
+
+    public void testDistanceTypeIsDeprecated() throws IOException {
+        String json =
+            "{\n" +
+                "  \"_geo_distance\" : {\n" +
```

Comment:
Shouldn't be `geo_distance`?

Reply:
I looked at that but the documentation says `_geo_distance` and then the code permits two misspellings.

Reply:
Could you point where in the documentation says that the two spellings are allowed? Looking in the code, a geo_distance query only allow `geo_distance` name.

Reply:
Sorry I didn't explain that well, those are 2 separate things:

Here is where we say (in the docs 0-9 through to 7.11) how you _should_ spell it in the DSL request:

https://www.elastic.co/guide/en/elasticsearch/reference/7.11/sort-search-results.html#geo-sorting

And here is where the tests were testing an alternate spelling

https://github.com/elastic/elasticsearch/blob/5eb0a9528a594b1bfff58be2da5a87bb7a161217/server/src/test/java/org/elasticsearch/search/sort/SortBuilderTests.java#L100

I agree it looks odd with the leading underscore

Reply:
Oh, hang on, my bad, sorry, I've just been looking at the Sorts this week, you are quite right, that should not have an underscore